### DISCUSSION TITLE
gasbormx

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#909] [Ab initio shape determination]
On 2012-11-09 10:23:27, user Marcus wrote:

Hi all,

I have a question concerning the new gasbormx since I wanted to try ab initio modeling on some oligomerising samples which always exist in two oligomeric states when measured with SAXS.
I was reading in the Acta paper on Atsas that you could tell the programm the how many percent of oligomer 1 and 2 there is in the sample. When I was trying to run it I did not find an option to tell gasbormx this with the knowledge I have from oligomer(I am using Atsas 2.5 on Linux).
Does the program try to find it out otherwise itself?

And maybe a second question: In the end it creates a pdb file with two chains. Can damaver keep this chain information or do you suggest not to run damaver after gasbormx

Cheers
Marcus

**Comment by franke-biosaxs** [comment]
On 2012-12-04 06:29:18, user SaxsMax wrote:

Hi Marcus,
Sorry for the delayed reply.
Indeed we have a version which allows one to fix the volume fractions,
but there were some issues with the convergence identified and it has been
decided to release the one where this feature is disabled.
As for damaver, the post-processing of Gasbormx models is not yet well-established.
What could be worth to do probably is to extract the monomers and
running Damaver with them. The resulting averaged model though is a model bead, not DR one anymore.

**Comment by franke-biosaxs** [comment]
On 2012-12-16 22:38:52, user jeremyk wrote:

Hi

I am interested in using GASBORMX for a oligomerising sample also.

However I am having a bit of trouble finding a manual (or paper) for running it, and I was wondering if I'm being a muppet in not being able to find it or if it is yet to be published?

Cheers

Jeremy

**Comment by franke-biosaxs** [comment]
On 2012-12-17 10:45:10, user SaxsMax wrote:

GasborMx has been described as a part of Atsas in the following paper
Petoukhov, M.V., Franke, D., Shkumatov, A.V., Tria, G., Kikhney, A.G., Gajda, M., Gorba, C., Mertens, H.D.T., Konarev, P.V. and Svergun, D.I. (2012)
New developments in the ATSAS program package for small-angle scattering data analysis. J. Appl. Cryst. 45, 342-350
(the top one on the page [http://www.embl-hamburg.de/biosaxs/publications.html](http://www.embl-hamburg.de/biosaxs/publications.html) )
There is no (yet) a paper specifically dedicated to the program.

Basically, you run it in the same way as you would do with the conventional Gasbor for the intact symmetric oligomer
(and GasborMx simply takes into account that some portion of the oligomers are dissociated into monomers)
